apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/evaluations/[evaluationId]/layout.tsx

```diff
@@ -94,15 +163,14 @@ export default async function ConnectedEvaluationLayout({
           />
         }
       />
-      <MetricsSummary
-        workspace={workspace}
+      <Content
         commit={commit}
         evaluation={evaluation}
         documentUuid={params.documentUuid}
-      />
-      <EvaluationResults
         evaluationResults={evaluationResults}
-        evaluation={evaluation}
+        isNumeric={isNumeric}
```

Comment:
you are already passing the evalution to the content component you don't need this derived state prop also

Reply:
I use to type the component with a generic